src/ui/styles/colors.scss

```diff
@@ -37,24 +37,12 @@
     #47ff94 119.14%
   );
 
-  --ion-color-gradient-w3c-card: linear-gradient(
+  --ion-color-gradient-card: linear-gradient(
```

Comment:
Not sure if we really need to keep both `--ion-color-gradient-card` and `--ion-color-gradient-acdc-credential.`

Is there any case where we still use the generic one?

Maybe `--ion-color-gradient-card` can be removed.